input/docs/handbook/collections/index.md

```diff
@@ -77,10 +77,30 @@ var myBindingOperation = mySource
 ```
 The API for the above is the same for cache and list.
 
+### Migrate from ReactiveList to DynamicData
+
+A starting point of migration from ReactiveList to DynamicData is to change the code of 
+
+```cs
+public ReactiveList<ItemViewModel> BindingData {get;set;}  //used for binding like ItemsSource="{Binding BindingData}"
+```
+
+to 
+
+```cs
+protected SourceList<ItemViewModel> mySource;
+public ReadOnlyObservableCollection<ItemViewModel> BindingData {get;} //used for binding like ItemsSource="{Binding BindingData}"
+...
+mySource.Connect()
```

Comment:
This example could just be as easily replaced with a observable collection. Not a great example without filtering or threading.